Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -176,9 +198,9 @@ echo disk > /sys/power/state
 		while ($sw.elapsed -lt $timeout){
 			$vmCount = $AllVMData.Count
 			Wait-Time -seconds 15
-			$state = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password "date"
+			$state = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password -command "date; echo $?"
```

Comment:
here it will return output of both commands like
```
#date; echo $?
Wed Jun 17 05:20:09 UTC 2020
0
```

Do you really want that? or you can redirect date output to /dev/null and then echo $?

Reply:
Fixed in e183f43c39346e5c4724e47ad1c718e7ad3391fc